### PR TITLE
Use cdn remove gulp

### PR DESCRIPTION
**Pull request by insacuri**
Branched off https://github.com/ONSdigital/ras-frontstage/pull/294 which needs to be merged in first.
There is A LOT of styling code that isn't used in this repo.  It makes it really confusing when figuring out what is actually being used.
This PR is at least a first pass, as I'm fairly sure there is more unused code that can be deleted, at removing unused code to make the codebase tidier.
This PR should have more then the normal amount of approval and each person reviewing it should test it to ensure that nothing looks obviously out of place.

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -36,6 +36,15 @@ class Config(object):
     RAS_FRONTSTAGE_API_SERVICE = '{}://{}:{}/'.format(RAS_FRONTSTAGE_API_PROTOCOL,
                                                       RAS_FRONTSTAGE_API_HOST,
                                                       RAS_FRONTSTAGE_API_PORT)
+
+    RAS_SECURE_MESSAGING_PROTOCOL = os.getenv('RAS_SECURE_MESSAGING_PROTOCOL', 'http')
+    RAS_SECURE_MESSAGING_HOST = os.getenv('RAS_SECURE_MESSAGING_HOST', 'localhost')
+    RAS_SECURE_MESSAGING_PORT = os.getenv('RAS_SECURE_MESSAGING_PORT', 5050)
+
+    RAS_SECURE_MESSAGING_SERVICE = '{}://{}:{}/'.format(RAS_SECURE_MESSAGING_PROTOCOL,
```
Comment (rhiGriff) [inline config.py]: These shoud be RAS_SECURE_MESSAGE_SERVICE_PORT ect, to coincide what's in Jenkins

**Comment by emilioward** [comment]
I don't know how to review this theres too many files that have been changed and I don't know which code is specific to the PR you have raised

**Comment by JamesGardiner** [comment]
Same as Emilio - if you branch off a branch, point the PR at that branch until the upstream is merged to master - it makes it easier to review 👍

**Comment by insacuri** [comment]
A v2 version of this branch now has a PR

**Pull request closed without merging**